test/unit/lodash.spec.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+
+describe('Lodash training', function ()
+{
+    describe('Number functions', function ()
+    {
+        describe('clamp', function ()
+        {
+
+            var params, number, lower, upper;
+
+            before(function ()
+            {
+                params = datasets.clamp();
+                number = params[0];
+                lower = params[1];
+                upper = params[2];
+            });
+
+            it('should verify params', function ()
+            {
+                expect(params).to.have.length(3);
+                expect(lower).to.be.below(upper);
+                expect(typeof number).to.eql('number');
+                expect(typeof lower).to.eql('number');
+                expect(typeof upper).to.eql('number');
+            });
+
+            it('should return the clamped number', function ()
+            {
+                expect(_.clamp.apply(_, params)).to.eql(3);
+            });
```

Comment:
When annotation about three describes will be applied, then we can merge those **it**s into one. I mean for each of three describes of course, so we will have three **it**s in fact but one per describe.

Texts of those **it**s have to be appropriate, so e.g. for first situation (describe where first param lower then second) **it** can be: *should return value equal second param*